console/queue_publisher.py

```diff
@@ -0,0 +1,68 @@
+import pika
```

Comment:
I'm going to ignore this for now - it should get swallowed up by sdx-common at some point